src/components/SharingButtons.jsx

```diff
@@ -0,0 +1,102 @@
+                </svg>
+              </div>
+            </div>
+          </a>
+        )
+      }
+
+      {/* linkedIn Sharing Button */}
+      if (this.props.linkedin) {
+        data.push(
+          <a key={"linkedin"} className="sharing-buttons__item__link" href={'https://www.linkedin.com/shareArticle?mini=true&amp;url=' + url + '&amp;title=' + title + '&amp;summary=' + teaser + '&amp;source=' + url} target="_blank" aria-label="">
+            <div className="sharing-buttons__item sharing-buttons__item--linkedin">
+              <div aria-hidden="true" className="sharing-buttons__item__icon">
+                <svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24">
+                  <path strokeWidth="1px" strokeLinejoin="round" strokeMiterlimit="10" d="M6.5 21.5h-5v-13h5v13zm-2.51-15h-.029c-1.511 0-2.488-1.182-2.488-2.481 0-1.329 1.008-2.412 2.547-2.412 1.541 0 2.488 1.118 2.519 2.447-.001 1.3-.978 2.446-2.549 2.446zm11.51 6c-1.105 0-2 .896-2 2v7h-5s.059-12 0-13h5v1.485s1.548-1.443 3.938-1.443c2.962 0 5.062 2.144 5.062 6.304v6.654h-5v-7c0-1.104-.896-2-2-2z" />
+                </svg>
+              </div>
+            </div>
+          </a>
+        )
+      }
+
+      {/* Email Sharing Button */}
+      if (this.props.email) {
+        data.push(
+          <a key={"email"} className="sharing-buttons__items__item__link" href={'mailto:?subject=The%20Gazelle%20%7C%20' + title + '&body=' + url} aria-label="">
+            <div className="sharing-buttons__item sharing-buttons__item--email">
+              <div aria-hidden="true" className="sharing-buttons__item__icon">
+                <svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24">
+                  <path strokeWidth="1px" strokeLinecap="round" strokeLinejoin="round" strokeMiterlimit="10" d="M23.5 18c0 .828-.672 1.5-1.5 1.5h-20c-.828 0-1.5-.672-1.5-1.5v-12c0-.829.672-1.5 1.5-1.5h20c.828 0 1.5.671 1.5 1.5v12zM20.5 8.5l-8.5 5.5-8.5-5.5M3.5 16l3.5-2M20.5 16l-3.5-2" />
+                </svg>
+              </div>
+            </div>
+          </a>
+        )
+      }
+
+      return data;
+    }
+
+    return (
+      <div className="sharing-buttons">
+        {renderButtons()}
+      </div>
+    )
+  }
+}
+
+// Allows specification for which buttons are wanted
+SharingButtons.propTypes = {
+  facebook: React.PropTypes.bool,
+  twitter: React.PropTypes.bool,
+  linkedin: React.PropTypes.bool,
+  email: React.PropTypes.bool,
+  url: React.PropTypes.string.isRequired,
+  title: React.PropTypes.string.isRequired,
+  teaser: React.PropTypes.string,
+}
+
+// By default, all buttons are rendered
+SharingButtons.defaultProps = {
+  facebook: true,
+  twitter: true,
+  linkedin: true,
+  email: true,
+  url: "http://www.thegazelle.org/",
+  title: "The Gazelle | NYUAD's Independent Student News Publication",
```

Comment:
Just check with editors that this is the ofificial line. Also, should maybe have Title and Teaser not be repeating each other.